test/certs/ca_config.txt

```diff
@@ -0,0 +1,17 @@
+# Copyright (C) 2020, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+[ req ]
```

Comment:
This cert stuff is used by the admission-controller. There should be no need to deploy the admission-controller.

Reply:
Verrazzano operator needs the cert also for system-tls and default-secxret.  I can try deleting them and see if it breaks the operator

Reply:
You are right,  I am not seeing errors if i delete system-tls and default-secret.  I have removed this.